spec/factories/endpoint.rb

```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
+  factory :endpoint, :class => "Endpoint" do
+  end
```

Comment:
how about adding some defaults for these? 
https://github.com/ManageIQ/manageiq/blob/41eb953e114901120916d1f454de2211cca89ae5/app/models/endpoint.rb#L7-L10